core/src/main/java/com/linecorp/armeria/server/ServiceConfig.java

```diff
@@ -145,9 +150,20 @@ public HttpService service() {
         return service;
     }
 
+    /**
+     * Returns the default value of the {@link RequestLog#serviceName()} property which is used when
+     * no service name was set via {@link RequestLogBuilder#name(String, String)}.
+     * If {@code null}, the path pattern of the {@link #route()} will be used instead.
+     */
+    @Nullable
+    public String defaultServiceName() {
+        return defaultServiceName;
+    }
+
     /**
      * Returns the default value of the {@link RequestLog#name()} property which is used when no name was set
-     * via {@link RequestLogBuilder#name(String)}. If {@code null}, HTTP method name will be used instead.
+     * via {@link RequestLogBuilder#name(String, String)}. If {@code null}, HTTP method name will be used
+     * instead.
```

Comment:
Question: Should we mention that some service types such as annotated services and RPC services will set the serviceName/name property automatically even if a user doesn't set them?